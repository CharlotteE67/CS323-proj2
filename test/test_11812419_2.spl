int test1(int x,float y){
    float x = 0.25;
}

int test1(char z){
    return 0;
}

int main(){

}