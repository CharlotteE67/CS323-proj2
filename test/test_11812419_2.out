Error type 3 at Line 2: variable redefined.
Error type 4 at Line 5: function redefined.